cts/cts-exec.in

```diff
@@ -499,7 +499,7 @@ class ExecTests(Tests):
                      + self._action_timeout)
 
         test.add_cmd(args='-l "NEW_EVENT event_type:exec_complete rsc_id:stonith_test_rsc action:monitor rc:error op_status:error" -t 15000',
-                     kill="killall -9 -q pacemaker-fenced lt-pacemaker-fenced")
+                     kill="killall -9 -q pacemaker-fenced lt-pacemaker-fenced", expected_exitcode=ExitStatus.TIMEOUT)
```

Comment:
It seems odd that we have rc:error op_status:error here but get a timeout. It's trying to run a monitor command after killing the fencer, so either one could make sense, but the inconsistency is odd.

Reply:
I thought this was odd too.  I can work on bisecting to figure out where this broke.

Reply:
Hah, figured it out.  This patch is completely unnecessary and the real fix is to change a couple "Error" into "Error occurred".